oss-common/src/main/kotlin/dev/ocpd/oss/FileStore.kt

```diff
@@ -73,6 +73,23 @@ interface FileStore {
      */
     fun exists(key: String): Boolean
 
+    /**
+     * Move a file from one key to another.
+     *
+     * @param source The key of the file to be moved.
+     * @param dest   The key of the file to be moved to.
+     * @return True if the file is moved successfully, false otherwise.
+     */
+    fun mv(source: String, dest: String): Boolean
```

Comment:
```suggestion
    fun move(source: String, dest: String): Boolean
```
Assuming `move` is not a keyword.